index.html

```diff
@@ -251,13 +251,15 @@ <h1> About<br> Myself</h1>
     <div class="contact">
         <h2>Contact me</h2>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
-        <form  action="https://formspree.io/f/xvoyrvqp"  method="POST">
+        
+        <form action="https://formspree.io/f/xvoyrvqp"  method="POST"  class="contact-me"  >
+            <div class="email-validation"></div>
```

Comment:
- [ ] Nice job. As part of the requirements, the error message should be displayed just on top of the submit button